libmscore/articulation.h

```diff
@@ -98,7 +98,7 @@ class Articulation final : public Element {
       virtual void write(XmlWriter& xml) const override;
       virtual bool readProperties(XmlReader&) override;
 
-      virtual QLineF dragAnchor() const override;
+      QVector<QLineF> dragAnchorLines() const override;
```

Comment:
I notice that you've removed `virtual` specifiers on member-function declarations that are declared as `override`, both here and elsewhere.

Is this a new official coding guideline?

I like having `virtual` there, even though it's technically redundant.

Reply:
No it doesn't defined by any coding guidelines in MuseScore right now. But maybe we should come to some single style here.

Somewhat in defense of this change, [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Inheritance) suggests not writing `virtual` in this case:
```
Explicitly annotate overrides of virtual functions or virtual destructors with exactly one of an override or (less frequently) final specifier. Do not use virtual when declaring an override.
```